README.md

```diff
@@ -1,6 +1,8 @@
 # buildah-build
 [![CI checks](https://github.com/redhat-actions/buildah-build/workflows/CI%20checks/badge.svg)](https://github.com/redhat-actions/buildah-build/actions?query=workflow%3A%22CI+checks%22)
-[![Verify Build](https://github.com/redhat-actions/buildah-build/workflows/Test%20Build/badge.svg)](https://github.com/redhat-actions/buildah-build/actions?query=workflow%3A%22Test+Build%22)
+[![Test Build with dockerfile](https://github.com/redhat-actions/buildah-build/workflows/Test%20Build%20with%20dockerfile/badge.svg)](https://github.com/redhat-actions/buildah-build/actions?query=workflow%3A%22Test+Build+with+dockerfile%22)
```

Comment:
Can you maybe shorten the workflow names. They will look too long on these badges

Reply:
What about `Dockerfile build` and `Scratch build` :thinking: 
or may me `Build from Scratch` & `Build from Dockerfile`
I know second option is not that short.

Reply:
how about `buildah` and `buildah bud`

Reply:
I don't think this clarifies the thing that we are achieving by this workflow.

Reply:
"Build" and "Build from Dockerfile" ?

Reply:
This is good :+1: